test/exports.js

```diff
@@ -0,0 +1,11 @@
+const { Headers, Request, Response, FetchError } = require('../')
+const fetch = require('minipass-fetch')
+const { test } = require('tap')
+
+test('exports the same helper classes', t => {
+  t.equal(Headers, fetch.Headers)
```

Comment:
The problem is that this would pass without the patch, because `undefined === undefined`.  It'd be better to do something like: `t.match(Headers, Function)` to verify that functions are being exported.